CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Operator `repeated` (#1026)
 
 ### Fixed
+
+- VSCode plugin: Highlighting of identifiers that contain numbers (#1028)
```

Comment:
There is a separate changelog for the vscode plugin (inside `quint-vscode`). Please move this to that changelog.

Reply:
I missed that. Fixed!